packages/interwovenkit-react/src/pages/deposit/SelectLocalAsset.module.css

```
@@ -27,14 +27,24 @@
+
 .asset:hover {
   background-color: var(--gray-8);
 }
 
-.asset img {
+.asset img,
+.imgPlaceholder {
   width: 32px;
   height: 32px;
 
   border: 1px solid var(--gray-5);
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)